src/ifd/ifd.rs

```diff
@@ -159,6 +159,20 @@ impl Ifd {
         self
     }
 
+    /// Returns the same `Ifd`, after adding the specified pairs of Tags and Values.
+    ///
+    /// Because it returns `Self`, it is possible to chain this method.
+    ///
+    pub fn with_entries(mut self, mut entries: Vec<(FieldTag, Box<FieldValues>)>) -> Self {
+        entries.drain(..).for_each(|(tag, value)| {
```

Comment:
Is there a reason you used `drain` instead of `into_iter`? 

Maybe this function could be implemented for any generic type implementing  IntoIterator instead of just Vec.

Reply:
Thanks for the tip!